src/SpotifyLogin.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
+export const SpotifyLogin = () => {
+    const initiateSpotifyLogin = () => {
+        const authUrl =
+            `https://accounts.spotify.com/authorize?` +
```

Comment:
it would be good to extract auth url generation into smith like utils/spotify.js to keep Spotify endpoints and api calls in a single place (for future functionality)